assets/css/education.css

```diff
@@ -1,28 +1,35 @@
 @import url(./style.css);
+@font-face {
+    font-family: "NHaas";
+    src: url(../fonts/NHaas\ Bold.63b9aba.woff2);
+}
 
 /* Dark and Light theme */
 
 html[data-theme='light'] {
     --bg1-color: #ecf0f3;
     --shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
     --border: 1px solid #ecf0f3;
+    --font-color: black;
 }
 
 html[data-theme='dark'] {
-    --bg1-color: #272727;
-    --shadow: 0 4px 8px 0 rgba(104, 102, 102, 0.2);
-    --border: none;
+    --bg1-color: #000033;
```

Comment:
Please have a look at the reference I gave, inspect the website and get the colors. The entire point of changing this is to make sure that the cards are visible separately from the background color